tests/pytorch/test_cuda_graphs.py

```diff
@@ -199,7 +201,7 @@ def _test_cuda_graphs(
 
     # Loss function and optimizer.
     if not dpa:
-        optimizer = optimizer(model.parameters(), lr=0.001)
+        optimizer = torch.optim.SGD(model.parameters(), lr=0.001)
```

Comment:
In dropping the adam optimizer from the test parameterization, are we losing any test coverage w.r.t. cuda graphs?

Reply:
This test doesn't really test convergence, but just checks that results match exactly with and without CUDA graphs. SGD and Adam are implemented similarly, so we just need to test one to make sure that the CUDA graph infrastructure is working correctly.